examples/temperature-measurement-app/esp32/main/main.cpp

```diff
@@ -75,14 +82,18 @@ static AppDeviceCallbacks EchoCallbacks;
 static void InitServer(intptr_t context)
 {
     Esp32AppServer::Init(); // Init ZCL Data Model and CHIP App Server AND Initialize device attestation config
+#if CONFIG_ENABLE_ESP_DIAGNOSTICS_TRACE
+    diagnosticStorage.Init(endUserBuffer, CONFIG_END_USER_BUFFER_SIZE);
```

Comment:
why are we passing `endUserBuffer` twice? first time in the constructor and second time here?